server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java

```diff
@@ -742,6 +749,9 @@ private synchronized void replayLogs(long targetRevision) {
 
                 updateLastReplayedRevision(nextRevision);
                 info.lastReplayedRevision = nextRevision;
+                if (command instanceof UpdateServerStatusCommand) {
```

Comment:
`command.type() == CommandType.UPDATE_SERVER_STATUS`

Reply:
`instanceof` would be better since:
- `command` is casted to `UpdateServerStatusCommand`
- command can be null. `instanceof` can remove an additional null check.